### PR TITLE
Issue3

### PR DESCRIPTION
**Pull request by tvirolai**
Basic implementation for create-project and drop-project.

### CODE REVIEW

File: annif/annif.py [hunk 1]
```diff
@@ -54,9 +55,24 @@ def listprojects():
...
+                }
+            }
+    results = es.search(index=annif.config['INDEX_NAME'], doc_type='project',
+            body=doc)
+    projects = [x['_source'] for x in results['hits']['hits']]
+    for p in projects:
+        print(p)
+
+    print(projects)
+    report = cat.indices()  # This queries different indices and returns
```
Comment (osma) [inline annif/annif.py]: Is this really necessary? The result seems to be just thrown away.

File: annif/annif.py [hunk 2]
```diff
@@ -54,9 +55,24 @@ def listprojects():
...
+                'match_all': {}
+                }
+            }
+    results = es.search(index=annif.config['INDEX_NAME'], doc_type='project',
+            body=doc)
+    projects = [x['_source'] for x in results['hits']['hits']]
+    for p in projects:
+        print(p)
+
+    print(projects)
```
Comment (osma) [inline annif/annif.py]: This is redundant after the previous for loop

Comment (tvirolai) [reply annif/annif.py]: Oops, true. I tried different ways to present the results when configuring tests. This I'll remove this.

File: annif/annif.py (hunk 1 above)
Comment (tvirolai) [reply annif/annif.py]: Not really, same thing applies here. I'll do another commit to remove these before merging.

**Pull request merged**